Jenkinsfile

```diff
@@ -81,20 +75,52 @@ pipeline {
         }
       }
     }
-    stage('Publish Release')
+    stage('Release Packages')
     {
       when {
           expression {
               "${env.SEMVER_STATUS}" == "0" && "${env.BRANCH_NAME}"  == "master"
           }
       }
+      agent {
+          docker {
+              image "rightbrainnetworks/auto-semver:${env.VERSION}"
+          }
+      }
       steps
       {
-        unstash 'PACKAGE'
         // Create GitHub Release & Upload Artifacts
         createGitHubRelease('rbn-opsGitHubToken', 'RightBrain-Networks/auto-semver', "${env.SEMVER_RESOLVED_VERSION}",
           "${env.SEMVER_RESOLVED_VERSION}", ["auto-semver.tar.gz" : "dist/semver-${env.SEMVER_NEW_VERSION}.tar.gz"])
 
+
+        // Upload package to PyPi
+        withCredentials([usernamePassword(credentialsId: env.PYPI_CREDENTIALS, usernameVariable: 'PYPI_USERNAME', passwordVariable: 'PYPI_PASSWORD')]) {
+          sh("pip install twine")
```

Comment:
can this be done in a docker agent? I'd prefer to not have packages installed on our jenkins.

Reply:
Within Docker image